Services/Randomization/service.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version = "1.0" encoding = "UTF-8"?>
+<service xmlns="http://www.w3.org" version="$Id$" id="rnd">
+	<logging />
```

Comment:
A new `ilCtrl` structure reload is req. for this.

Reply:
Yes, I know. Wanted to bypass conflicts until this PR is approved.